Quantum-Control-Applications/Quantum-Dots/Use Case 1 - Fast 2D Scans/README.md

```diff
@@ -0,0 +1,110 @@
+
+|  ![Raster_Scan](./_images/Raster_Scan.png)  |  ![Raster_Waveform](./_images/Raster_Waveform.png)  |
+|:-------------------------------------------:|:---------------------------------------------------:|
+
+In the above images, the voltage from analog output 1 controls the x coordinate in the image, and analog output 2 controls the y. 
+Analog output 3 is responsible for creating the RF tone for measurement. The output voltages have been offset by 1.1V for clarity. 
+
+Through this method I personally routinely performed 100x100 pixel scans in 5ms, where each pixel was measured for a total of 500ns. 
+However, I discovered a flaw in the methodology, which I then circumvented using the OPX. 
+
+### The Flaw in Raster Scans
+
+My setup for performing the fast measurements was to connect the AWG to the gates through the capacitive node of a bias tee. 
+Thus, introducing a high pass filtering effect on the signals sent from the AWG. In our setup the time constant for this 
+filtering effect was found to be approximately 1ms, orders of magnitude away from being an issue for qubit manipulation. 
+
+However, for the fast scans it was quite prohibitive, as it places hard limits of the resolution and/or the time spent 
+at each pixel. 
+indeed, the period of the slower sawtooth was given by the time spent per pixel multiplied by the total number of pixels 
+in the image.  
+When this period is comparable to the time constant of the filter, we found that the two-dimensional scans 
+were distorted and difficult to interpret. 
+It was possible, up to a certain point, to pre-compensate the slower sawtooth for the distortion of the high pass filter, 
+allowing us to perform our 100x100 scan with a time per pixel of 1us meaning that the period of the slow sawtooth was 10ms. 
+
+### Moving Away From Raster
+
+The programability of the OPX made it possible to move away from Raster scans, thus greatly avoiding this issue. 
+Rather than Raster scans we used a spiral pattern to navigate to every pixel across the two-dimensional window. 
+
+|  ![Spiral_Scan](./_images/Spiral_Scan.png) | ![Spiral_Waveform](./_images/Spiral_Waveform.png)  |
+|-------------------------------------------:|:---------------------------------------------------|
+
+For a scan of $N \times N$  pixels where time $\tau$ is spent at each pixel, the slowest frequency needed for the spiral 
+pattern is $f=1/(2N\tau)$ whereas for a Raster scan it was $f=1/(N^2\tau)$.
+
+Thus allowing much greater resolutions and/or time spent at each pixel before the high pass filtering effect becomes an issue. 
+Shown below is the effect of a high pass filter with a time constant of 200µs on the waveforms needed to create 101x101 raster and spiral scans with a measurement time of 1us. 
```

Comment:
The figure below is not very clear (to those who don't know it).
I would add a few words describing it...
I would also recreate the figures without analog output 3 (it just takes away focus, give out no information) and would rename the captions:
Ideal V_rg
Ideal V_lg
Simulated filtered V_rg
Simulated filtered V_lg

Reply:
Very good point, thanks !
I tried to improve it so please tell me if something remains unclear